File: Q1a.py
sum=0
nofdist=0
for i in range(1,7):
    print("--o--",end='')#displaying grid
    for j in range(1,7):
        sum+=abs(i-j)#calculating sum of all distances and no. of distances
        nofdist+=1
print("\nThe average distance between two points is: ",sum/nofdist)

File: Q1a.cpp
#include<iostream>
using namespace std;
int main(){
float sum=0;
float nofdist=0;
    for (int i=1;i<=6;i++){
        cout<<"--o--";//displaying the grid
        for(int j=1;j<=6;j++){
            sum+=abs(i-j);//calculating sum of all distances and no. of distances
            nofdist+=1;
        }
    }
cout<<endl<<"The average distance between two points is: "<<sum/nofdist;
return 0;
}
